client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/pivot/HistogramGroupSource.java

```diff
@@ -97,4 +97,28 @@ public boolean equals(Object other) {
     public int hashCode() {
         return Objects.hash(field, interval);
     }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+
+        private String field;
+        private double interval;
+
+        public Builder setField(String field) {
+            this.field = field;
+            return this;
+        }
+
+        public Builder setInterval(double interval) {
```

Comment:
Same question, should we use "Duration" here?

Reply:
that does not make sense here as `interval` in this case is any arbitrary number. Any numeric field can be grouped via a histogram, not just a time field.

Reply:
A, ok. So this was my misunderstanding (I associated "interval" with some time duration).